rules/linux/process_creation/proc_creation_lnx_device_driver_insert_remove.yml

```
@@ -0,0 +1,28 @@
+title: Detect Adding Kernel Modules - Linux
+id: 85b78882-a58d-405f-bbb3-201d41812e8c
+status: test
```

```suggestion
status: experimental
```
See https://github.com/SigmaHQ/sigma-specification/blob/main/specification/sigma-rules-specification.md#status
And https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq-rule-convention.md#status
Thanks